src/ContentImporter.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+namespace Drupal\openy;
+
+use Drupal\migrate\MigrateMessage;
+use Drupal\migrate\Plugin\Migration;
+use Drupal\migrate_tools\MigrateExecutable;
+
+/**
+ * Class ContentImporter.
+ *
+ * @package Drupal\openy
+ */
+class ContentImporter implements ContentImporterInterface {
+
+  /**
+   * Migration manager.
+   *
+   * @var \Drupal\migrate_plus\Plugin\MigrationConfigEntityPluginManager
+   */
+  protected $migrationManager;
+
+  /**
+   * Map of content items and migrations.
+   *
+   * @var array
+   */
+  protected $map;
+
+  /**
+   * ContentImporter constructor.
+   */
+  public function __construct() {
+    $this->map = $this->getMap();
+
+    // Service container are not available on the first steps of profile installation.
+    $this->migrationManager = \Drupal::service('plugin.manager.config_entity_migration');
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getMap() {
+    // @todo Move the map into config.
+    // @todo Set dependencies for "landing" and "programs".
+    return [
+      'landing' => [],
+      'branches' => [
+        'openy_demo_node_branch'
+      ],
+      'blog' => [
+        'openy_demo_node_blog',
+      ],
+      'programs' => [],
+    ];
+  }
```

Comment:
Should we also trigger there taxonomy terms or they will be triggered automatically?